src/CommitmentsV2/SFA.DAS.CommitmentsV2/Application/Queries/GetCohortSummary/GetCohortSummaryQueryHandler.cs

```diff
@@ -46,7 +46,8 @@ public async Task<GetCohortSummaryQueryResult> Handle(GetCohortSummaryQuery requ
                     LastUpdatedByEmployerEmail = c.LastUpdatedByEmployerEmail,
                     LastUpdatedByProviderEmail = c.LastUpdatedByProviderEmail,
  	 				IsApprovedByEmployer = c.EditStatus == EditStatus.Both || (c.Apprenticeships.Any() && c.Apprenticeships.All(a => a.AgreementStatus == AgreementStatus.EmployerAgreed || a.AgreementStatus == AgreementStatus.BothAgreed)),
-                    IsApprovedByProvider = c.EditStatus == EditStatus.Both || (c.Apprenticeships.Any() && c.Apprenticeships.All(a => a.AgreementStatus == AgreementStatus.ProviderAgreed || a.AgreementStatus == AgreementStatus.BothAgreed))
+                    IsApprovedByProvider = c.EditStatus == EditStatus.Both || (c.Apprenticeships.Any() && c.Apprenticeships.All(a => a.AgreementStatus == AgreementStatus.ProviderAgreed || a.AgreementStatus == AgreementStatus.BothAgreed)),
+                    EmployerCanApprove = !c.Apprenticeships.Any(a => a.FirstName == null || a.LastName == null || a.DateOfBirth == null || a.CourseName == null || a.StartDate == null || a.EndDate == null || a.Cost == null)
```

Comment:
It looks like this doesn't cater for empty cohorts - if there are no draft apprenticeships, the cohort is also said to be not complete.